lib/Parser/Parse.cpp

```diff
@@ -5374,18 +5392,8 @@ void Parser::ParseFncDeclHelper(ParseNodeFnc * pnodeFnc, LPCOLESTR pNameHint, us
             {
                 m_reparsingLambdaParams = true;
             }
-            DeferredFunctionStub *saveDeferredStub = nullptr;
```

Comment:
Do we no longer need this save-and-restore? I remember we had issues with function bodies in default expressions. Is that no longer the case?

Reply:
Yes, these are still an issue, I've added back this save-and-restore.